src/managed/CommonManaged.props

```diff
@@ -11,6 +11,8 @@
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <IncludeSymbols>true</IncludeSymbols>
     <Serviceable>true</Serviceable>
+    <PackageLicenseUrl>https://github.com/dotnet/core-setup/blob/master/LICENSE.TXT</PackageLicenseUrl>
```

Comment:
yea - I just found that.  Nevermind.  What you have here is good.

I was looking at what we shipped for 2.0.0-preview1.  Looks like that has changed since.